azure-pipelines.yml

```diff
@@ -110,7 +110,7 @@ stages:
 
 - stage: DeploySit
   displayName: 'Deploy to Sit'
-  condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'develop'))
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
```

Comment:
only the develop branch should be deployed to SIT

Reply:
After had call with John , agreed to push this change to allow the testing on SIT